.github/workflows/build.yml

```diff
@@ -35,6 +35,38 @@ jobs:
           SANDBOX_TOKEN: ${{ secrets.API_HUB_SANDBOX_TOKEN }}
       - run: yarn test:integration
       - run: yarn test:type
+  tests-windows:
+    runs-on: ${{ matrix.os }}
+    continue-on-error: true
+    timeout-minutes: 15
+    strategy:
+      matrix:
+        os: [windows-latest]
+        node-version: [12.x]
+    steps:
+      - uses: actions/checkout@v2
+      - run: git fetch --depth=1
+      - uses: actions/setup-node@v2
+        with:
+          node-version: ${{ matrix.node-version }}
+      - uses: actions/cache@v2
+        id: cache
+        with:
+          path: |
+            node_modules
+            */*/node_modules
+          key: ${{ matrix.os }}-${{ hashFiles('**/yarn.lock') }}
+      - run: yarn install --frozen-lockfile
+      - run: yarn test:unit -- -- --no-bail -- -- -- --json --outputFile=unit-test-output.json
+      - run: yarn test:integration --json --outputFile=int-test-output.json
+        if: always()
+      - run: yarn test:type
+      - name: Archive results
+        uses: actions/upload-artifact@v2
+        if: failure()
+        with:
+          name: windows-test-results
+          path: '*packages/**/*-test-output.json'
```

Comment:
can you explain how this writes the output the the packages? I did not find the documentation on this.

Reply:
Adding command options `--outputFile=unit-test-output.json` to jest outputs the test results inside each respective workspace where tests ran. Related documentation is [here](https://docs.github.com/en/actions/advanced-guides/storing-workflow-data-as-artifacts#example)